microprofile.jdt/pom.xml

```diff
@@ -38,6 +39,10 @@
       <layout>p2</layout>
       <url>https://download.eclipse.org/jdtls/milestones/1.15.0/repository/</url>
     </repository>
+    <repository>
+      <id>jdt.ls.maven</id>
+      <url>https://repo.eclipse.org/content/repositories/jdtls-releases/</url>
```

Comment:
Changes this to `jdt-snapshots` . Releases wouldn't have 1.16.0-SNAPSHOT.

Reply:
Also, I to get it to work locally, I needed to change the `id` to something like `jdt.ls.maven2` :\ which seems odd, but it does work afterwards.

Reply:
Looks like it works for me! Didn't even need to change the id

Reply:
Thanks for the help!